tests/run-make/windows-safeseh/rmake.rs

```diff
@@ -0,0 +1,15 @@
+//@ only-windows
+//@ needs-rust-lld
+
+use run_make_support::rustc;
+
+fn main() {
+    // Ensure that LLD can link when an .rlib contains a synthetic object
+    // file referencing exported or used symbols.
+    rustc().input("foo.rs").arg("-Clinker=rust-lld").run();
```

Comment:
Question: should we add a `linker()` helper method to `Rustc`? Something simple like

```rs
impl Rustc {
    fn linker(&mut self, linker: &str) -> &mut Self {
        self.cmd.arg(format!("-Clinker={linker}"));
        self
    }
}
```

Reply:
Makes sense to me!